package/kernel/linux/files/sysctl-nf-conntrack.conf

```diff
@@ -4,5 +4,5 @@
 net.netfilter.nf_conntrack_acct=1
 net.netfilter.nf_conntrack_checksum=0
 net.netfilter.nf_conntrack_tcp_timeout_established=7440
-net.netfilter.nf_conntrack_udp_timeout=60
-net.netfilter.nf_conntrack_udp_timeout_stream=180
+net.netfilter.nf_conntrack_udp_timeout=300
+net.netfilter.nf_conntrack_udp_timeout_stream=60
```

Comment:
Why do you change this to 60?
What does this value have to do with 9bf6078866b29e9dc89374c79cc750390e73f95c? It was added in 629e73938eae1add67c2cbfdd62672f529422045.

Reply:
It was added exactly vice versa from old parameter. Also unreplied status is triggered by single packet.

Reply:
The "stream" state happens as long as "request" like DNS or NTP or SNMP is not yet replied,

Reply:
Default is nf_conntrack_udp_timeout=30 and nf_conntrack_udp_timeout_stream=120, not the other way around.
https://www.kernel.org/doc/Documentation/networking/nf_conntrack-sysctl.txt

Reply:
What protocols wait for longer than a minute for UDP response? Super ancient libc may wait 15s for dns response.

Reply:
There is a blatant falsiety in kernel documentation
> 
nf_conntrack_udp_timeout_stream - INTEGER (seconds)
	default 120

	This extended timeout will be used in case there is an UDP stream
	detected.

There is absolutely no way "stream is detected" without time travel as in other news (nft manual page) it is stated that replied status cannot be cleared, and in yet another section about setting timeouts it is missed out that udp states have 2 timeouts - unreplied and replied.

You can set both to 300s if you believe in time travel though.

Reply:
So you are having issues with YouTube/Quic and you can reproduce it, right? Which of these two parameters fixes the problem, nf_conntrack_udp_timeout or nf_conntrack_udp_timeout_stream? Why change the other if only one of them is the problem? Can you make a reproducer for us (without watching hours of YouTube shorts)? For my users, YouTube is fine.

Reply:
The timeout without stream alone fixes the problem.
Repeater cgi for http/3 quic
```
echo "<html>"
sleep 190
echo "</html>"